config/test-spider-gps-imu.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": 2,
+  "robot": {
+    "drivers": [ "spider", "gps", "imu" ],
+    "spider": {
+        "port": "/dev/ttyS0", "speed": 115200, "stream_id_in": 1, "stream_id_out": 2
+    },
+    "gps": {
```

Comment:
Just a note for the future - with this format it will not be possible to instantiate two drivers of the same type since the `robot` is dictionary and keys are driver names.